Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Project.java

```diff
@@ -721,4 +802,76 @@ public int hashCode() {
         return this.title == null ? 0 : this.title.hashCode();
     }
 
+    /**
+     * Returns the non-deactivated production templates associated with the
+     * project.
+     *
+     * @return the active production templates
+     */
+    public List<Template> getActiveTemplates() {
+        if (Objects.isNull(templates)) {
+            return null;
```

Comment:
Would it not be better instead of returning `null` here returning an empty list?

Reply:
Good catch, needs fixing.